_includes/integrations/angular-quick-start.md

```diff
@@ -80,33 +80,33 @@ This procedure requires:
       >
     </editor>
     ```
-    This TinyMCE editor configuration should replicate the example on the [Basic example page]({{site.baseurl}}/demo/basic-example/).
-7. Provide access to TinyMCE using Tiny Cloud or by self-hosting TinyMCE.
+    This {{site.productname}} editor configuration should replicate the example on the [Basic example page]({{site.baseurl}}/demo/basic-example/).
+7. Provide access to {{site.productname}} using {{site.cloudname}} or by self-hosting {{site.productname}}.
 
-    * **Tiny Cloud**
+    * **{{site.cloudname}}**
 
-        Include the `apiKey` option in the editor element and include your [TinyMCE API key]({{site.shared_baseurl}}/signup/).
+        Include the `apiKey` option in the editor element and include your [{{site.accountpage}} API key]({{site.shared_baseurl}}/signup/).
```

Comment:
Another place where the wrong variable may have been used?

Reply:
fair, changed to `cloudname`